docs/book/GLOSSARY.md

```diff
@@ -146,6 +164,12 @@ Unless otherwise specified, this refers to horizontal scaling.
 
 The infrastructure that backs a [Machine Resource](#user-content-machine), typically either a cloud instance, virtual machine, or physical host.
 
+# T
+
+## Target cluster
+
+The declared cluster we intend to create and manage using Bootstrap cluster.
```

Comment:
If pivot is redefined to remove the hard reference to the bootstrap cluster, then this definition will need to be modified as well.

Reply:
Hi @detiber ,thanks for replying.
I agree with you on the pivot definition changes but confused regarding this change. I may be wrong but is it not that the term "target cluster" is used only in the case where we are using a bootstrap cluster. Because there need to be a difference between target and a workload cluster.

Reply:
If describing the `clusterctl create` workflow it is true that you have bootstrap cluster and a target cluster, but in the context of just pivot the source and target clusters can be any pre-existing Kubernetes cluster. It gets even more complicated if discussing the `clusterctl delete` workflow, since the source cluster for pivot is the remote cluster and the target cluster is the "bootstrap" cluster.

Reply:
Ohh okay. I will update it. Thanks @detiber